client/src/components/layout/ReviewTile.js

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+
+
+const ReviewTile = (props) => {
+
+    const {title, reviewContent, rating} = props
```

Comment:
you can do this destructuring inline on line 4
`const ReviewTile = ({title, reviewContent, rating}) => {`